OpenRA.Game/Actor.cs

```diff
@@ -202,6 +222,11 @@ public IEnumerable<T> TraitsImplementing<T>()
 			return World.TraitDict.WithInterface<T>(this);
 		}
 
+		public T TraitInfo<T>()
```

Comment:
The problem is that the distinction between an Actor/Trait and the ActorInfo/TraitInfo is subtle  (see my comment in #7565), and many people have issues distinguishing them. Adding a method here to query directly from an actor to its info's TraitInfos is only going to make this worse.

Demeter's law doesn't really apply because the Actor/ActorInfo and Trait/TraitInfo are closely related by design, and really form the core of simulation engine.